src/content/profile-data.js

```diff
@@ -66,9 +80,10 @@ export function getFuncStackInfo(stackTable: StackTable, frameTable: FrameTable,
       }
     }
 
+    // Go through each stack, and create a new funcStack table, which is based off of
+    // functions rather than frames.
     for (let stackIndex = 0; stackIndex < stackTable.length; stackIndex++) {
       const prefixStack = stackTable.prefix[stackIndex];
-      // assert(prefixStack === null || prefixStack < stackIndex);
```

Comment:
Maybe just rephrase this as `// We know that at this point the following condition holds: ...`